### ISSUE TITLE
rule: nontable-no-empty-cells

### ISSUE THREAD

**Issue by purupurupu** [purupurupu/css-noop-checker#129]
## Rule ID
`nontable-no-empty-cells`

## What does this rule detect?
`empty-cells` only applies to elements with `display: table-cell`. On any other display type, it has no effect.

## Example: should warn
```html
<div style="empty-cells: hide;">content</div>
```

## Example: should NOT warn
```html
<td style="empty-cells: hide;"></td>
```

## Related CSS Properties
`display`, `empty-cells`

## References
- [CSS 2.2 §17.6.1.1 - empty-cells](https://www.w3.org/TR/CSS22/tables.html#empty-cells): applies to table-cell elements
- [Firefox inactive-property-helper.js](https://searchfox.org/mozilla-central/source/devtools/server/actors/utils/inactive-property-helper.js) — detects this pattern
